cms/djangoapps/contentstore/views/course.py

```diff
@@ -467,6 +470,54 @@ def _get_rerun_link_for_item(course_key):
     return reverse_course_url('course_rerun_handler', course_key)
 
 
+def _ora1_deprecation_info(course_id, advanced_modules):
+    """
+    Returns information about ORA1
+        * Modules presence in the Advanced Module List.
+        * Components presence in the course outline.
+
+    Arguments:
+        course_id: course id
+        advanced_modules: advance module list
+
+    Returns:
+        Dict with following keys:
+
+        ora1_enabled (bool): True if "peergrading" and/or "combinedopenended" are in the Advanced Module List else False
+        ora1_components (list): List of ora1 components and their parent's url
+        advance_settings_url (str): URL to advance settings page
+    """
+    modified_timestamp = CourseStructure.objects.filter(course_id=course_id).values('modified')
+    if modified_timestamp.exists():
+        cache_key = 'ora1.components.{course}.{modified}'.format(
+            course=course_id,
+            modified=modified_timestamp[0]['modified']
+        )
+    else:
+        return {}
```

Comment:
Can we make default dict like this ->
```
data = {
'ora1_enabled': False,
'ora1_components': [],
'advance_settings_url': ''
}
```
and return data instead of `{}`.